src/workers/personalizedDigestEmail.ts

```diff
@@ -166,6 +192,55 @@ const worker: Worker = {
       return;
     }
 
+    const analyticsQueue = fastq.promise(
+      async (
+        data: {
+          user_id: string;
+          device_id: string;
+          experiment_id: string;
+          variation_id: number;
+        } & AnalyticsEvent,
+      ) => {
+        await sendAnalyticsEvent([data]);
+      },
+      1,
+    );
+
+    const growthbookClient = getUserGrowthBookInstace(user.id, {
+      enableDevMode: process.env.NODE_ENV !== 'production',
+      subscribeToChanges: false,
```

Comment:
Recommended for request/expiring runtimes like worker/request by [gb docs](https://docs.growthbook.io/lib/js#:~:text=Set%20this%20to%20%60false%60%20to%20improve%20performance%20in%20server%2Dside%20environments).